tests/conftest.py

```diff
@@ -190,8 +190,15 @@ def pytest_terminal_summary(terminalreporter, exitstatus, config):
                 println("Removing missing tests from record.")
             println("")
 
+    if ui_option == "test" and _should_write_ui_report(exitstatus):
+        println("\n-------- Suggested fixtures.json diff: --------")
+        ui_tests.write_fixtures_suggestion(config.getoption("ui_check_missing"))
```

Comment:
this is super confusing. I'd suggest printing just the filename here, and doing the "write suggestions file" step outside `pytest_terminal_summary`

Reply:
fa6aa4e67bfcd454e20c73af988cc2440593953f